examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py

```diff
@@ -56,49 +55,80 @@
 global_batch = local_batch * strategy.num_replicas_in_sync
 base_lr = 0.007 * global_batch / 16
 
+# TODO(tanzhenyu): add a diff dataset.
+# all_ds = load(split="sbd_train", data_dir=None)
+# all_ds = all_ds.concatenate(load(split="sbd_eval", data_dir=None))
+# train_ds = all_ds.take(10000)
+# eval_ds = all_ds.skip(10000).concatenate(load(split="diff", data_dir=None))
 train_ds = load(split="sbd_train", data_dir=None)
 eval_ds = load(split="sbd_eval", data_dir=None)
 
-resize_layer = tf.keras.layers.Resizing(512, 512)
+resize_layer = tf.keras.layers.Resizing(512, 512, interpolation="nearest")
```

Comment:
Should we use standard bilinear interpolation for the image, even though we need to use "nearest" for the segmask? (I guess maybe it doesn't really matter?)

Reply:
It probably doesn't matter, but if we don't choose the same method, then merging this to KPL will cause an issue?